docker/Dockerfile.base_e2e

```diff
@@ -7,25 +7,22 @@ ENV TZ=Etc/UTC
 
 # Install common tools/git/software-properties-common
 RUN apt-get update \
-    && apt-get install -y ca-certificates curl gnupg lsb-release wget jq vim rsync zip unzip\
-    && apt-get install -y git git-lfs\
+    && apt-get install -y ca-certificates curl gnupg lsb-release wget jq vim rsync zip unzip git git-lfs sudo \
     && git lfs install \
     && apt-get install -y software-properties-common
 
 # Install docker
 RUN mkdir -p /etc/apt/keyrings \
     && curl -fsSL https://download.docker.com/linux/ubuntu/gpg | gpg --dearmor -o /etc/apt/keyrings/docker.gpg \
     &&  echo \
-         "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
-         $(lsb_release -cs) stable" | tee /etc/apt/sources.list.d/docker.list > /dev/null \
+    "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+    $(lsb_release -cs) stable" | tee /etc/apt/sources.list.d/docker.list > /dev/null \
     && apt-get update \
     && apt-get install -y docker-ce-cli
 
 # Install kubectl/helm
-RUN curl -fsSLo /usr/share/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg \
-    && echo "deb [signed-by=/usr/share/keyrings/kubernetes-archive-keyring.gpg] https://apt.kubernetes.io/ kubernetes-xenial main" | tee /etc/apt/sources.list.d/kubernetes.list \
-    && apt-get update \
-    && apt-get install -y kubectl \
+RUN curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl" \
+    && sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl \
```

Comment:
Running cmd under the root user does not need `sudo`

Reply:
fixed